Misc/NEWS.d/next/C_API/2025-09-07-10-52-09.gh-issue-60107.dhcb2Y.rst

```diff
@@ -0,0 +1 @@
+* gh-60107: avoid io.RawIOBase.read from reading into a temporary bytearray (author: taskevich)
```

Comment:
```suggestion
Avoid :meth:`io.RawIOBase.read` from reading into a temporary :class:`bytearray`
when calling its own :meth:`io.RawIOBase.readinto` internally.
```

And move this blurb entry to `Library` instead of `C API`.